packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -442,6 +452,35 @@ class InspectorService extends InspectorServiceBase {
     );
   }
 
+  Future<void> _addPubRootDirectories(List<String> pubDirectories) {
+    // No need to call this from a breakpoint.
+    assert(useDaemonApi);
+    return invokeServiceMethodDaemonNoGroupArgs(
+      'addPubRootDirectories',
+      pubDirectories,
+    );
+  }
+
+  Future<void> _removePubRootDirectories(List<String> pubDirectories) {
+    // No need to call this from a breakpoint.
+    assert(useDaemonApi);
+    return invokeServiceMethodDaemonNoGroupArgs(
+      'removePubRootDirectories',
+      pubDirectories,
+    );
+  }
+
+  Future<List<String>?> getPubRootDirectories() async {
+    assert(useDaemonApi);
+    final response = await invokeServiceMethodDaemonNoGroupArgs(
+      'getPubRootDirectories',
```

Comment:
these will all fail on the bots if we do not bump DevTools to a version of Flutter that your flutter change is included in